scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -34,7 +34,7 @@
     'fileUploadFeatures', 'library', 'navigation', 'playVoiceovers',
     'preferences', 'profileFeatures', 'profileMenu', 'publication',
     'subscriptions', 'topicsAndSkillsDashboard', 'topicAndStoryEditor',
-    'topicAndStoryEditorFileUploadFeatures', 'users']
+    'topicAndStoryEditorFileUploadFeatures', 'users', 'fileUploadExtensions']
```

Comment:
This is supposed to be sorted, but you can do that in a follow up PR.

Reply:
done, fixed it